jira_agile_metrics/calculators/impediments_test.py

```diff
@@ -155,13 +155,13 @@ def test_calculate_impediments(query_manager, settings, cycle_time_results):
 def test_different_backlog_column(query_manager, settings, cycle_time_results):
     settings = extend_dict(settings, {
         'backlog_column': 'Committed',
+        'committed_column': 'Build',
```

Comment:
Why did you change this test?

Feels like this might mean changing previous behaviour?

If not and you just want to test a different scenario, could we leave this test and add a new one?

Reply:
As described in https://github.com/DeloitteDigitalUK/jira-agile-metrics/pull/27#issuecomment-636308642 I think that `impediments.py` should only consider `active_columns`, i.e. from `committed_column` up to and excluding `done_column`.